packages/block-library/src/image/image.js

```diff
@@ -481,11 +482,13 @@ export default function Image( {
 						</ToolsPanelItem>
 					) }
 					{ isResizable && dimensionsControl }
-					<ResolutionTool
-						value={ sizeSlug }
-						onChange={ updateImage }
-						options={ imageSizeOptions }
-					/>
+					{ !! imageSizeOptions.length && (
+						<ResolutionTool
+							value={ sizeSlug }
+							onChange={ updateImage }
+							options={ imageSizeOptions }
+						/>
+					) }
```

Comment:
This doesn't seem to make a difference in terms of the actual behaviour, or am I missing something?

Reply:
Oooh I see, no it's fine to leave it here! Both are related to reset behaviour.